### PR TITLE
Update profile information, contact information, and volunteer type forms

### PR DESCRIPTION
**Pull request by joyceycodes**
### ProfileInformationForm.tsx, ContactInformationForms.tsx, VolunteerTypeForm.tsx**

- Added component state, on change event handler, and form submission handler 

### CreateUserPassForm.tsx

- Fixed bug in handleChange() function

### Signup.tsx

- Updated file documentation
- Changed "is_student":boolean property --> "volunteer_type": string. It should be easier to keep track of the 3 different types of users (general, student, healthcare provider) using this property instead compared to a boolean. **_(FOR ARI)_** 
- Added further conditions render the correct forms based on the signupData state

### CODE REVIEW

**Comment by syangyue** [comment]
Nice job!

**Review by syangyue — DISMISSED** [review]
So cool

File: src/components/ContactInformationForm.tsx
Comment (syangyue) [inline src/components/ContactInformationForm.tsx:1]: amzaing!

**Review by syangyue — APPROVED** [review]
Amazing!

**Review by ari-denary — APPROVED** [review]

**Pull request merged**